go.mod

```diff
@@ -9,8 +9,9 @@ require (
 	cosmossdk.io/depinject v1.0.0-alpha.3
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.4
-	cosmossdk.io/store v0.1.0
+	cosmossdk.io/store v0.0.0-20230126202154-be9bd7a8c1b4
```

Comment:
why this change? sorry if I'm missing any context here?

Reply:
this happened while doing some changes related to cosmovisor which are now reverted.
But we can leave these changes this way as it just uses a commit versioning which resulting in no replace of store